<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiUserInfo</class>
 <widget class="QMainWindow" name="GuiUserInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>User Infomation</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="lId">
            <property name="text">
             <string>Id:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lName">
            <property name="text">
             <string>Full Name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lCode">
            <property name="text">
             <string>Code:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lJob">
            <property name="text">
             <string>Job:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lEmail">
            <property name="text">
             <string>Email:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lDob">
            <property name="text">
             <string>Date of Birth:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="listInput">
          <item>
           <widget class="QLineEdit" name="iUserId"/>
          </item>
          <item>
           <widget class="QLineEdit" name="iUserName"/>
          </item>
          <item>
           <widget class="QLineEdit" name="iUserCode"/>
          </item>
          <item>
           <widget class="QLineEdit" name="iUserJob"/>
          </item>
          <item>
           <widget class="QLineEdit" name="iUserEmail"/>
          </item>
          <item>
           <widget class="QDateEdit" name="iUserDoB">
            <property name="displayFormat">
             <string>dd/MM/yyyy</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bOK">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
